src/components/NonNimiqCheckoutOption.vue

```diff
@@ -201,6 +231,68 @@ export default class NonNimiqCheckoutOption<
 </script>
 
 <style scoped>
+    .currency-info h1 {
+        display: flex;
+        align-items: center;
+        margin-bottom: 0;
+    }
+
+    .currency-info img,
+    .currency-info .nq-icon {
+        width: 4.5rem;
+        height: 4.5rem;
+        margin-right: 2rem;
+    }
+
+    .payment-option .small-page {
+        position: relative;
+        width: 52.5rem;
+    }
+
+    .payment-option .nq-card-wrapper {
+        position: relative;
+    }
+
+    .payment-option.confirmed .nq-card-wrapper {
+        perspective: 250rem;
+        --safari-rotate-fix: translateZ(1px);
+    }
+
+    .nq-card-wrapper > .transition-flip-enter-active,
```

Comment:
`transition-flip` can move to `App.vue` at some point, just like `transition-fade`.